crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -80,11 +80,13 @@ pub(crate) async fn farm(
     .await??;
 
     let multi_farming = MultiFarming::new(
-        base_directory,
-        client,
-        object_mappings.clone(),
-        reward_address,
-        best_block_number_check_interval,
+        multi_farming::Options {
```

Comment:
I'd import `Options` instead of call it more verbose if you're concerned with generic name

Reply:
I don't see how it would solve the problem if I would call it `MultiFarmingOptions`. It is still would be generic. Main difference would be in importing vs not importing.

I think that calling things with generic names inside the module and using them with this name is fine, as we know the context. Overall it is fine as it can be referred using the module name like here.

Reply:
Then rename during import. I'm just trying to have more consistency here. We typically import data structures unless there are some conflicts or other reasons to not import. I just don't see why all of a sudden explicit path is needed here.